manual/io/index.md

```diff
@@ -298,7 +298,7 @@ Most analyses on the other hand will benefit from one of the fast compression al
 The compression factor, that is, the saving of storage space, varies with the type of data. A buffer containing `N` identical values is compressed better than a set of values with higher entropy.
 
 ROOT offers [several options](https://root.cern/doc/master/structROOT_1_1RCompressionSetting_1_1EAlgorithm.html#ad5ed4ef81888f5406ae0f018ff45fd96), such as LZMA with very high compression ratio, or LZ4 with very high decompression throughput, or ZSTD with a good compromise in performance.
-The default compression for [`RNTuple`](https://root.cern/doc/master/structROOT_1_1Experimental_1_1RNTuple.html) is determined based on the data; for everything else it's zlib with compression level 1.
+The default compression for [`RNTuple`](https://root.cern/doc/master/structROOT_1_1Experimental_1_1RNTuple.html) is ZSTD level 5; for everything else it's zlib with compression level 1.
```

Comment:
While we're at it, this link should be updated to `https://root.cern/doc/master/classROOT_1_1RNTuple.html` following the move from `Experimental` (and it's a `class` already since a while)